src/main/java/wooteco/subway/domain/DiscountRole.java

```diff
@@ -0,0 +1,45 @@
+package wooteco.subway.domain;
+
+import java.util.Arrays;
+import java.util.function.Predicate;
+import wooteco.subway.domain.strategy.discount.AdolescentDiscountStrategy;
+import wooteco.subway.domain.strategy.discount.DiscountStrategy;
+import wooteco.subway.domain.strategy.discount.FreeDiscountStrategy;
+import wooteco.subway.domain.strategy.discount.KidDiscountStrategy;
+import wooteco.subway.domain.strategy.discount.NonDiscountStrategy;
+
+public enum DiscountRole {
+
+    BABY(age -> age < Constants.KID_STANDARD, new FreeDiscountStrategy()),
+    KID(age -> age >= Constants.KID_STANDARD && age < Constants.ADOLESCENT_STANDARD,
+            new KidDiscountStrategy()),
+    ADOLESCENT(age -> age >= Constants.ADOLESCENT_STANDARD && age < Constants.ADULT_STANDARD,
+            new AdolescentDiscountStrategy()),
+    ADULT(age -> age > Constants.ADULT_STANDARD && age < Constants.SENIOR_STANDARD,
+            new NonDiscountStrategy()),
+    SENIOR(age -> age >= Constants.SENIOR_STANDARD, new FreeDiscountStrategy());
+
+    private static class Constants {
+        private static final int KID_STANDARD = 6;
+        private static final int ADOLESCENT_STANDARD = 13;
+        private static final int ADULT_STANDARD = 19;
+        private static final int SENIOR_STANDARD = 65;
+    }
+
+    private Predicate<Integer> predicate;
+    private DiscountStrategy strategy;
+
+    public static DiscountStrategy findDiscountStrategy(int age) {
+        return Arrays.stream(values()).filter(role -> role.predicate.test(age)).findAny()
+                .orElseThrow(() -> new IllegalArgumentException("전략을 찾을 수 없습니다.")).getStrategy();
```

Comment:
```suggestion
Arrays.stream(values())
                .filter(role -> role.predicate.test(age))
                .findAny()
                .orElseThrow(() -> new IllegalArgumentException("전략을 찾을 수 없습니다."))
                .getStrategy();
```
체이닝마다 개행을 넣어주면 가독성 측면에서 좋을거 같아요.

Reply:
지난 번에 받았던 피드백인데, 아직 습관으로 형성이 안되어 놓치고 있네요. 의식하여 개행하겠습니다. 죄송합니다 🥲